packages/ui-library/src/components/form-caption-group/index.stories.ts

```diff
@@ -36,11 +36,11 @@ Technically both instances of the Form Caption are placed inside slots that are
 The Form Caption Group is intended to be used when creating new components. Currently, it is used like this in the following components:
         
 - [**Checkbox**](?path=/docs/components-checkbox--docs)
-- [**Number Input**](?path=/docs/components-number-input--docs)
+- [**Input Field Number**](?path=/docs/components-input-field-number--docs)
 - [**Radio**](?path=/docs/components-radio--docs)
 - [**Radio Group**](?path=/docs/components-radio-group--docs)
 - [**Select**](?path=/docs/components-select--docs)
-- [**Text Input**](?path=/docs/components-text-input--docs)
+- [**Input Field Text**](?path=/docs/components-input-field-text--docs)
```

Comment:
Could you move this up in between line 39 and 40? So the components are sorted alphabetically?